test/framework/src/main/java/org/elasticsearch/cluster/routing/TestShardRouting.java

```diff
@@ -297,16 +300,17 @@ public static UnassignedInfo randomUnassignedInfo(String message) {
             if (randomBoolean()) {
                 delayed = true;
             }
-            lastAllocatedNodeId = randomAlphaOfLength(10);
+            lastAllocatedNodeId = randomIdentifier();
         }
         int failedAllocations = reason == UnassignedInfo.Reason.ALLOCATION_FAILED ? 1 : 0;
+        long unassignedTimeNanos = randomNonNegativeLong();
```

Comment:
There's a CI test hitting this. You may fix the test, or do a random time here in the past (like a random time before the year 2000 if it works).

Reply:
Yeah, probably it is the best way to do it. Not sure if it is concerning that we might have shards that are unassigned years before